src/EnergyPlus/AirLoopHVACDOAS.cc

```diff
@@ -1008,9 +1007,7 @@ namespace AirLoopHVACDOAS {
             state.dataLoopNodes->Node(this->m_FanOutletNodeNum).MassFlowRateMaxAvail = sizingMassFlow;
             state.dataLoopNodes->Node(this->m_FanOutletNodeNum).MassFlowRateMax = sizingMassFlow;
         }
-        if (errorsFound) {
-            ShowFatalError(state, "Preceding sizing errors cause program termination");
-        }
+
         state.dataSize->CurSysNum = state.dataHVACGlobal->NumPrimaryAirSys + this->m_AirLoopDOASNum + 1;
```

Comment:
[src/EnergyPlus/AirLoopHVACDOAS.cc:1011]:(style),[knownConditionTrueFalse],Condition 'errorsFound' is always false